<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>插补计算</string>
  </property>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>10</y>
     <width>141</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="placeholderText">
    <string>请输入1-100的整数</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>步长:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>20</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_3">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>50</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>起点x坐标</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>50</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>起点y坐标</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_4">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>20</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>20</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>终点x坐标</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>50</y>
     <width>69</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>终点y坐标</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_5">
   <property name="geometry">
    <rect>
     <x>520</x>
     <y>50</y>
     <width>61</width>
     <height>25</height>
    </rect>
   </property>
   <property name="clearButtonEnabled">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>80</y>
     <width>91</width>
     <height>71</height>
    </rect>
   </property>
   <property name="title">
    <string>插补方式</string>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="checked">
    <bool>false</bool>
   </property>
   <widget class="QRadioButton" name="radioButton">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>直线插补</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>118</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>圆弧插补</string>
    </property>
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox_2">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>120</width>
     <height>80</height>
    </rect>
   </property>
   <property name="title">
    <string>圆弧插补方向</string>
   </property>
   <widget class="QRadioButton" name="radioButton_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>20</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>顺时针插补</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioButton_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>50</y>
      <width>95</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>逆时针插补</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>lineEdit</sender>
   <signal>editingFinished()</signal>
   <receiver>Widget</receiver>
   <slot>out_step()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>186</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>721</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_2</sender>
   <signal>editingFinished()</signal>
   <receiver>Widget</receiver>
   <slot>on_lineEdit_2_editingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>421</x>
     <y>30</y>
    </hint>
    <hint type="destinationlabel">
     <x>586</x>
     <y>175</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_3</sender>
   <signal>editingFinished()</signal>
   <receiver>Widget</receiver>
   <slot>on_lineEdit_3_editingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>435</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>604</x>
     <y>132</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_4</sender>
   <signal>editingFinished()</signal>
   <receiver>Widget</receiver>
   <slot>on_lineEdit_4_editingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>26</y>
    </hint>
    <hint type="destinationlabel">
     <x>602</x>
     <y>108</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>lineEdit_5</sender>
   <signal>editingFinished()</signal>
   <receiver>Widget</receiver>
   <slot>on_lineEdit_5_editingFinished()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>543</x>
     <y>63</y>
    </hint>
    <hint type="destinationlabel">
     <x>609</x>
     <y>245</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radioButton</sender>
   <signal>toggled(bool)</signal>
   <receiver>Widget</receiver>
   <slot>slot1()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>98</x>
     <y>111</y>
    </hint>
    <hint type="destinationlabel">
     <x>299</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>out_step()</slot>
  <slot>on_lineEdit_2_editingFinished()</slot>
  <slot>on_lineEdit_3_editingFinished()</slot>
  <slot>on_lineEdit_4_editingFinished()</slot>
  <slot>on_lineEdit_5_editingFinished()</slot>
  <slot>slot1()</slot>
 </slots>
</ui>
